libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -16,34 +16,57 @@ export type ButtonType =
     templateUrl: './button.component.html',
     styleUrls: ['./button.component.scss']
 })
-export class ButtonComponent extends BaseComponent {
-    /** Option to make to button compact. */
-    @Input()
-    compact: boolean;
+export class ButtonComponent extends BaseComponent implements AfterViewInit {
 
-    /** The icon to include in the button */
+    /** The icon to include in the button. See the icon page for the list of icons.
+     * Setter is used to control when css class have to be rebuilded.
+     * Default value is set to ''.
+     */
     @Input()
-    glyph: string;
+    public glyph: string;
```

Comment:
no need to add public, it is public by default

Reply:
done